backend/FwLite/FwLiteShared/Services/MiniLcmJsInvokable.cs

```diff
@@ -11,9 +11,10 @@ public class MiniLcmJsInvokable(
     IMiniLcmApi api,
     BackgroundSyncService backgroundSyncService,
     IProjectIdentifier project,
-    MiniLcmApiNotifyWrapperFactory apiWrapperFactory) : IDisposable
+    MiniLcmApiNotifyWrapperFactory apiWrapperFactory,
```

Comment:
this parameter should have a more descriptive name now that it's not the only wrapper

Reply:
Done in commit 27c607d0d.